### ISSUE TITLE
Making it work

### ISSUE THREAD

**Issue by Chiny91** [KeepThisTicket/weatherClock#2]
Congratulations on achieving [recognition](https://www.raspberrypi.org/blog/raspberry-pi-weatherclock-shows-you-the-hours-forecast/) with this project - Raspberry Pi .org no less.   I ordered the Pimoroni display last night when they had 11 in stock; less than 24 hours later, no stock 😱

A few things slowed my installation of this software on my fresh Raspbian Buster on an R Pi Zero, which may help those that follow:

Get the weatherClock software:
`git clone https://github.com/KeepThisTicket/weatherClock`

Force the use of Python 3 (or the default Raspbian may use the deprecated Python 2) AND change working directory.
Add to the top of weatherClock.py:
```
#!/usr/bin/python3
import os
os.chdir("/home/YOUR_USER/weatherClock/")
```

Install the Python tkinter module (don't use pip3 for this module):
`sudo apt-get install python3-tk`

My Pimoroni display has just been posted so whilst waiting, I have diverted the output from this software to my iMac using the XQuartz X Window system.   It works a treat and I do like the look of it, including the _touch for details_.   I suspect this will have a high [WAF](https://en.wikipedia.org/wiki/Wife_acceptance_factor) and I'll be printing a case shortly.

Many thanks for your efforts in making this project public.

**Comment by Vacant0mens** [comment]
@Chiny91 what's the purpose of changing directories? The script seems fairly self-contained (other than needing to install the `requests` module).

**Comment by Chiny91** [comment]
Not tried that code but it looks like it addresses the issue; script/image files in the same directory (likely if git clone is used).

My display has now arrived and very nice it looks, with weatherClock running fine.   I do notice that when I click on the hourly weather symbols (as of Thu 1pm), I am getting the weather forecast for at least 12 hours away (Fri 1am etc) rather than for later this afternoon (Thu 2pm 3pm etc).

Had to Google "POP", obvious when you know.   Here in the UK high summer, it is of course reading 100% 🌧️

**Comment by Vacant0mens** [comment]
I have a local branch that has that fix in it (among other things). Just waiting for approval to publish and make a PR.

**Comment by KeepThisTicket** [comment]
Hi guys sorry about the long silences - i have been busy and away the last few weeks but getting back into the swing of things now. @Vacant0mens is this confirmed resolved with the commit you have referenced above ?

**Comment by Vacant0mens** [comment]
yes, that commit is in [my PR](https://github.com/KeepThisTicket/weatherClock/pull/5) and should be good to go for this issue. 👍

**Comment by KeepThisTicket** [comment]
Closing issue as all should be resolved with work of @Vacant0mens with PR#8
